src/nodes/display/GaussianBlurNode.js

```diff
@@ -155,7 +155,8 @@ class GaussianBlurNode extends TempNode {
 		//
 
 		const material = this._material || ( this._material = builder.createNodeMaterial() );
-		material.fragmentNode = blur();
+		material.fragmentNode = blur().context( builder.getSharedContext() );
```

Comment:
> There are possibly more nodes to add the .context( builder.getSharedContext() )

Regarding the FX passes, it should be at least `AnamorphicNode` and `AfterImageNode`.

Do you mind explaining why `.context( builder.getSharedContext() )` is now used in `GaussianBlurNode` (and not just `RTTNode`)?

Reply:
As it is a different render-pass, this shader is compiled in a different `NodeBuilder` and will probably not share the same context or the same settings defined in the renderer, any Node that uses the RenderToTexture approach can inherit the context defined in its essence if it is transferred with this approach.

Reply:
So if this change would be missing in `GaussianBlurNode`, renderer settings could get out of sync and produce similar color errors like reported in GaussianBlurNode.

Reply:
> So if this change would be missing in GaussianBlurNode, renderer settings could get out of sync and produce similar color errors like reported in https://github.com/mrdoob/three.js/pull/28781#issuecomment-2208844881?

Yes, that's it, but only when a node uses context to obtain parameters like the `renderOutput()` case, in other situations it should be ok.